client/coral-admin/src/containers/ModerationQueue/ModerationContainer.js

```diff
@@ -90,6 +91,15 @@ class ModerationContainer extends Component {
     key.unbind('t');
   }
 
+  componentDidUpdate(_, prevState) {
+
+    // If paging through using keybaord shortcuts, scroll the page to keep the selected
+    // comment in view.
+    if (prevState.selectedIndex !== this.state.selectedIndex) {
+      document.querySelector(`.${styles.selected}`).scrollIntoViewIfNeeded();
```

Comment:
While `scrollIntoViewIfNeeded()` is admittedly totally awesome and I didn't know about it, [the docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded) say it's not on a standards track and doesn't work most places. We might just have to write some code to check the position in the window.